src/components/apartments/apartment/Apartment.css

```diff
@@ -8,13 +8,33 @@
 
 .photoContainer {
   height: 58px;
+  box-shadow: rgba(60, 64, 67, 0.3) 0px 1px 2px 0px, rgba(60, 64, 67, 0.15) 0px 2px 6px 2px;
 }
 
 .apartment-photo {
+ 
   width: 116px;
   height: 100%;
   background-size: cover;
   background-repeat: no-repeat;
+  z-index: -1;
+}
+
+.apartment-photo .apartment-unit {
```

Comment:
This selector is the in the wrong css file. Only put classes that apear in `Apartment.js` in this file. This selector belongs in `MiniImage.css`